src/style.css

```diff
@@ -0,0 +1,30 @@
+body {
+  margin: 0;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+}
+
+[class^="stars--"] {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars__star {
+  background: url(./images/star.svg) no-repeat center;
+  margin: 0 2px;
+  width: 16px;
+  height: 16px;
+}
+
+.stars__star:first-child {
+  margin-left: 0;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.active {
```

Comment:
The task requires that the number of active stars be different for all 6 blocks depending on the modifier class. Create separate classes for each block with the active stars, such as `.stars.stars--0 .stars__star:nth-child(-n+0)`, `.stars.stars--1 .stars__star:nth-child(-n+1)`, and so on.